usr/sbin/p11sak/p11sak.c

```diff
@@ -1356,6 +1365,8 @@ static CK_RV tok_key_get_key_type(CK_SESSION_HANDLE session, CK_OBJECT_HANDLE hk
                 ret);
         return ret;
     }
+
+
```

Comment:
did you add these 2 blank lines by intention? 1 blank line should be sufficient.....

Reply:
No intention. Removed one blank line.